tdrs-backend/tdpservice/users/api/login_redirect_oidc.py

```diff
@@ -8,13 +10,16 @@
 from django.http import HttpResponseRedirect
 from django.views.generic.base import RedirectView
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
Is it possible to standardize this with our past logging env var as shown [here](https://github.com/raft-tech/TANF-app/pull/1457/files#diff-15c50d3b3787aa9aa9caa3e31080db56148912a3ac2e1f23e39f3f884bbe2b7c)? I recall we ran into an issue when we were doing remote development so we discarded it then to make progress on this ticket.

```suggestion
logger = logging.getLogger(__name__)

```